### ISSUE TITLE
Max cpu switch

### ISSUE THREAD

**Issue by simon-p-r** [mpareja/paprika#10]
Hi

I cannot pass /m or /m:4 switch to module.

Nice module!

**Comment by mpareja** [comment]
Have you tried the "extraParameters" setting?

**Comment by simon-p-r** [comment]
Yes but I have now tried as an array and it appears to work, You may want to make changes to example to show passing an array rather than string.

```
 extraParameters: ['/p:OutDir=<OUTPATH>', '/m:4']
```

**Comment by simon-p-r** [comment]
OutDir and cpu switches are not working, I cannot pass alternative configuration to module.
